projects/aca-content/viewer/src/lib/components/preview/preview.component.spec.ts

```diff
@@ -56,12 +49,7 @@ describe('PreviewComponent', () => {
 
   beforeEach(() => {
     TestBed.configureTestingModule({
-      imports: [LibTestingModule, AcaViewerModule],
-      providers: [
-        { provide: DocumentBasePageService, useValue: DocumentBasePageServiceMock },
```

Comment:
I see one problem - if you check AppTestingModule then it contains some mock for rest services but for DocumentBasePageService it has: 
```ts
{ provide: DocumentBasePageService, useExisting: ContentManagementService }
```
which means it uses real (not mocked) service for this one service after you removed mock for that from these tests. We should not use real services for tests, we should to use mocks.